### PR TITLE
obs: enable fedora 29 

### PR DESCRIPTION
**Pull request by jcvenegas**
Add changes needed to enable Fedora 29

\cc @chavafg @marcov @lsm5

### CODE REVIEW

File: obs-packaging/distros [hunk 1]
```diff
@@ -8,6 +8,7 @@ CentOS_7::CentOS:CentOS-7::standard
 Fedora_26::Fedora:26::standard
 Fedora_27::Fedora:27::standard
 Fedora_28::Fedora:28::standard
+Fedora_29::Fedora:29::standard
```
Comment (marcov) [inline obs-packaging/distros]: I wasn't aware of this file. We'll need to add Debian 9.0 here too.

**Comment by marcov** [comment]
lgtm

**Comment by jodh-intel** [comment]
lgtm

File: obs-packaging/distros (hunk 1 above)
Comment (amshinde) [reply obs-packaging/distros]: @jcvenegas We should probably remove Fedora 26 and 27 from this list, since they have reached EOL.

Comment (jodh-intel) [reply obs-packaging/distros]: +1. Raised #279 to deal with that.

**Pull request merged**